electrochemistry.ttl

```diff
@@ -6446,6 +6935,114 @@ of other configurations are used."""@en ;
                                                        skos:prefLabel "LithiumManganeseOxideLithiumIronPhosphateElectrode"@en .
 
 
+###  http://emmo.info/emmo#EMMO_06448f64_8db6_4304_8b2c_e785dba82044
+emmo:EMMO_06448f64_8db6_4304_8b2c_e785dba82044 rdf:type owl:Class .
```

Comment:
Im not sure I follow these additions. Some are materials, but some are quantities (e.g. `Density`, `Thickness`). Shouldnt these be part of `electrochemicalquantities.ttl`?

Reply:
Thanks, you are right. I think this is some kind of bug in the export from Protege.